docs/proposals/formatted-output.md

```diff
@@ -0,0 +1,312 @@
+  "artifactType": "application/vnd.unknown.artifact.v1",
+  "config": {
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2,
+    "data": "e30="
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.oci.image.layer.v1.tar",
+      "digest": "sha256:6cb759c4296e67e35b0367f3c0f51dfdb776a0c99a45f39d0476e43d82696d65",
+      "size": 14477,
+      "annotations": {
+        "org.opencontainers.image.title": "sbom.spdx"
+      }
+    },
+    {
+      "mediaType": "application/vnd.oci.image.layer.v1.tar",
+      "digest": "sha256:54c0e84503c8790e03afe34bfc05a5ce45c933430cfd9c5f8a99d2c89f1f1b69",
+      "size": 6639,
+      "annotations": {
+        "org.opencontainers.image.title": "scan-test-verify-image.json"
+      }
+    }
+  ],
+  "annotations": {
+    "org.opencontainers.image.created": "2023-12-15T09:41:54Z"
+  }
+}
+```
+
+> [!NOTE]
+> The formatted output of `oras manifest fetch` only supports OCI/Docker Image Manifest in this release. [OCI image index](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc5/image-index.md) and other media types of artifact will be supported in future releases.
+
+## Scenarios
+
+### Scripting
+
+Alice is a developer who wants to batch operations with ORAS in her Shell script. In order to automate a portion of her workflow, she would like to obtain the image digest from the JSON output objects produced by the `oras push` command and then use shell variables or utilities like [xargs](https://en.wikipedia.org/wiki/Xargs) or  to enable an ORAS command to act on the output of another command and perform further steps. In this way, she can chain commands together. For example, she can use `oras attach` to attach an SBOM to the image using its image digest as a argument outputted from `oras push`.
+
+For example, push an artifact to a registry and generate the artifact reference in the standard output. Then, attach an SBOM to the artifact using the artifact reference (`$REGISTRY/$REPO@$DIGEST`) outputted from the first command. Finally, sign the attached SBOM with another tool against the reference of the SBOM file (`$REGISTRY/$REPO@$DIGEST`) that was obtained in the proceeding step.
+
+- Use shell variables on Unix
+
+```bash
+REFERENCE_A=$(oras push $REGISTRY/$REPO:$TAG hello.txt --format '{{.Ref}}')
+REFERENCE_B=$(oras attach --artifact-type sbom/example $REFERENCE_A sbom.spdx --format '{{.Ref}}') 
+notation sign $REFERENCE_B
+```
+
+- Use [ConvertFrom-Json](https://learn.microsoft.com/powershell/module/microsoft.powershell.utility/convertfrom-json) on Windows PowerShell
+
+```powershell
+$REFERENCE_A=oras push $REGISTRY/$REPO:$TAG hello.txt --format json --no-tty | ConvertFrom-Json
+$REFERENCE_B=oras attach --artifact-type sbom/example $REFERENCE_A.Ref sbom.spdx --format json --no-tty | ConvertFrom-Json
+notation sign $REFERENCE_B.Ref
```

Comment:
NIT: `ConvertFrom-Json` will produce an object containing the reference, suggest below changes 
```suggestion
$A=oras push $REGISTRY/$REPO:$TAG hello.txt --format json --no-tty | ConvertFrom-Json
$B=oras attach --artifact-type sbom/example $A.Ref sbom.spdx --format json --no-tty | ConvertFrom-Json
notation sign $B.Ref
```